hexa/files/gcp.py

```diff
@@ -248,7 +249,12 @@ def list_bucket_objects(
         objects = []
 
         def is_object_match_query(obj):
-            if ignore_hidden_files and obj["name"].startswith("."):
+            is_hidden = (
+                obj["name"].startswith(".")
```

Comment:
This one is covered by the last case no ? 
```
.hidden
a/b/c/.hidden
a/.b/c/hidden
.a/b/c/hidden
```

Reply:
Yes, I've added  it to handle hidden files in the root dir ( when ignore_delimiter=True and the file key`.file.ext`)